src/include/optimizer/child_property_generator.h

```diff
@@ -52,6 +54,11 @@ class ChildPropertyGenerator : public OperatorVisitor {
   void Visit(const PhysicalAggregate *) override;
 
  private:
+  /***** Helper functions *****/
```

Comment:
SeqScan and IndexScan have very similar behavior when deriving the child properties. Thus, we just take them out as a helper function to avoid duplicate code. The newest commit should have all the comments